code-challenges.test.js

```diff
@@ -14,13 +14,56 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("newArray", () => {
+  it("function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.", () => {
+    expect(newArray(fibLength1)).toEqual([1, 1, 2, 3, 5, 8]);
+    expect(newArray(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55,]);
+  });
+});
+
+
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+//● array1 › takes in an object and returns an array of the numbers sorted from least to greatest
+
+//ReferenceError: array1 is not defined
+
+// Test Suites: 1 failed, 1 total
+// Tests:       2 failed, 2 total
+// Snapshots:   0 total
+// Time:        0.47 s
+// Ran all test suites.
+
 // b) Create the function that makes the test pass.
+// create a function named fibOrder
+// create variables, num1 and num2 with a new value
+// add in a new variable called fibAdd to hold num1 + num2
+// need a new but empty array, newFibArr
+// for loop for our inptu number to iterate 
+// using .push to push num1 into the empty array 
+// return newly filled array newfibArr
+
+const fibOrder = (fibLength1) => {
+  let num1 = 1;
+  let num2 = 1;
+  let newFibArr = [];
+
+  for (let i = 0; i < fibLength1; i++) {
+    let fibAdd = num1 + num2
+    newFibArr.push(num1);
+    num1 = num2;
+    num2 = fibAdd;
+
+  }
+  return newFibArr
+}
+
+
```

Comment:
Great way to break this problem down to each step!  That's developer thinking!  As a stretch - I encourage you to refactor this using HOFs.  Awesome work, Christian!